examples/green_screen/README.md

```diff
@@ -0,0 +1,56 @@
+# Green Screen Example
+
+The goal of the green screen example is to demonstrate best practices for using multiple Azure Kinect devices, with an emphasis on synchronization and calibration (the 'green screen' code is only a small portion of the logic). In particular, the green screen application showcases a physical limitation of the hardware and how it can be mostly addressed using another device.
+
+## What does this application *do*?
+
+The green screen example displays the scene as observed by one of the cameras (the 'main' camera). Using the camera's depth, it will paint over anything beyond its depth threshold with green, making it appear as if only the things in the foreground are there. It will fill in missing details with the 'backup' camera, if possible, resulting in a better green screen than the main camera could achieve alone.
+
+## Why use two cameras? Isn't one good enough?
+
+It's true that one camera can get you most of the way to a good solution in this case. However, if you only use one camera (either by using the single-camera mode, or just covering the backup camera), and something in the scene is closer to the camera than something else in the scene (for example, if you hold out an arm), you should see a "shadow" of green pixels on the object in the background near the edge of the obstructing object.
```

Comment:
I appreciate adding code that demonstrates how to calibrate multiple K4A devices. I am not so sure if the green screen application is the best use case for a multi-Eden setup. The transformation_depth_to_color function should do a relatively good job in handling occlusions and it seems excessive to use multiple K4A devices to tackle this issue. A stronger example would be to show the combined point cloud in a 3d-viewer. As I have said, I also appreciate the current example. Thanks!

Reply:
An example for a multi kinect setup with point cloud registration for a combined point cloud would be super appreciated.
Thats exactly the setup we would like to have at my university.

Reply:
depth_to_color does do a fairly good job, but the difference between using the secondary camera and not using it is very noticeable when holding an arm out. It's true that the example does not have much utility outside of this specific use case and a 3d point cloud, in retrospect, may have been an overall more useful example.

However, part of the goal of this example was to demonstrate how to do something useful that required (or at least made effective use of) multiple cameras with minimal additional complexity, and this example does succeed in demonstrating how to, for example, synchronize multiple cameras, and how to calibrate them, and there is minimal code that doesn't pertain to either of those functions (this was one of the concerns with a point cloud example: that people not interested in the point cloud side of things would have to sift through that info).

I do agree that it would be great to have a combined 3d point cloud example. Knowing what we know now, there is definitely a strong demand for an example like that and it would be a great addition to the ecosystem. Unfortunately, I will probably be unable to produce something like that because I'm an intern and I'll be reaching the end of my time here very soon.

At the very least, the camera synchronization and camera calibration sections of this example should be relevant to the point cloud use case, even if doesn't cover everything. I hope it can still be useful in any projects you might be working on!

Reply:
Surely I appreciate this example as well.
We will definetly check this out for the calibration/synchronization aspect.
Thanks for your great effort. The commit history alone speaks about how much time was spent figuering all this out.